src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-size;
+  padding: 10px;
+  gap: $gap-size;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+  cursor: pointer;
+
+  .calendar__day-box {
+    background-color: #eee;
+    width: $box-size;
+    height: $box-size;
+    border: 1px solid black;
+    box-sizing: border-box;
+    text-align: center;
+    line-height: $box-size;
+    transition: transform 0.5s, background-color pink;
```

Comment:
something is wrong with the way of setting transition here, please read examples here - https://www.w3schools.com/css/css3_transitions.asp

`transition: transition-property transition-duration transition-timing-function`